cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -356,6 +356,9 @@ if(asimage)
   find_Package(PNG)
   if(PNG_FOUND)
     set(ASEXTRA_LIBRARIES ${ASEXTRA_LIBRARIES} ${PNG_LIBRARIES})
+    # Some missing variables needed for external PNG build
+    set(PNG_LIBRARY_RELEASE ${PNG_LIBRARY})
```

Comment:
Who's using `PNG_LIBRARY_RELEASE`? I see no usage in all of ROOT?

Reply:
It is a bit of "jungles" so when passing non-system png and printing cached CMake variables, I did it to be sure that all that there are no traces of system png library... (if you use system png library, value will be the same).